Document/0x05g-Testing-Network-Communication.md

```diff
@@ -15,6 +50,76 @@ Make sure that the hostname and the certificate itself are verified correctly. E
 
 ### Static Analysis
 
+#### Trust Anchors in the Network Security Configuration
+
+When running on Android 7.0 (API level 24) or higher, apps targeting those API levels will use a default Network Security Configuration that doesn't trust any user supplied CAs, reducing the possibility of MITM attacks by luring users to install malicious CAs. However, this protection can be bypassed by using a custom Network Security Configuration with a custom trust anchor indicating that the app will trust user supplied CAs.
+
+Use a decompiler (e.g. jadx or apktool) to confirm the target SDK version. After decoding the app you can look for the presence of `targetSDK` present in the file apktool.yml that was created in the output folder.
```

Comment:
```suggestion
Use a decompiler (e.g. jadx or apktool) to confirm the target SDK version. After decoding the app you can look for the presence of `targetSDK` in the file apktool.yml that was created in the output folder.
```

Reply:
apktool technically is a disassembler though, not a decompiler.

Reply:
This was one block I simply moved around I didn't see that so thanks for telling! We have to change it. I assume it just want to say: unpack and check that value in the android manifest.

> This is one of many cases where having a "Techniques" chapter we can refer back to will come handy. This way we don't have to explain that you need to unzip the APK, decode the Android Manifest..but you can achieve the same by ...blah blah We'd simply have "Checking a value from the Android Manifest" including all alternatives. And here we'd just say: `["Check that the value minsdk is ... in the Android Manifest"](linktotechniques#checking-a-value...)`.